### PR TITLE
add fr tour default parameter values

### PR DESCRIPTION
**Pull request by apointeau**

### CODE REVIEW

File: _fr/tour/default-parameter-values.md [hunk 1]
```diff
@@ -3,10 +3,48 @@ layout: tour
...
+Cela peut être utilisé pour omettre ces paramètres lors des appels aux fonctions.
+
+```scala mdoc
+def log(message: String, level: String = "INFO") = println(s"$level: $message")
+
+log("System starting")  // prints INFO: System starting
+log("User not found", "WARNING")  // prints WARNING: User not found
+```
+
+Le paramètre `level` a une valeur par défaut, donc il est optionnel. Sur la dernière ligne, l'argument `"WARNING"` écrase la valeur par défaut `"INFO"`. Là où vous pourriez "overload" les méthodes en Java, vous pouvez simplement utiliser les paramètres optionnels pour obtenir le même effet. Cependant, lorsque l'appel ignore un argument, tous les arguments suivants doivent être nommés.
```
Comment (julienrf) [inline _fr/tour/default-parameter-values.md:24]:
```suggestion
Le paramètre `level` a une valeur par défaut, donc il est optionnel. Sur la dernière ligne, l'argument `"WARNING"` écrase la valeur par défaut `"INFO"`. Là où vous pourriez surcharger les méthodes en Java, vous pouvez simplement utiliser les paramètres optionnels pour obtenir le même effet. Cependant, lorsque l'appel omet un argument, tous les arguments suivants doivent être nommés.
```

File: _fr/tour/default-parameter-values.md [hunk 2]
```diff
@@ -3,10 +3,48 @@ layout: tour
...
+
+```scala mdoc
+class Point(val x: Double = 0, val y: Double = 0)
+
+val point1 = new Point(y = 1)
+```
+
+Ici nous pouvons dire que `y = 1`.
```
Comment (julienrf) [inline _fr/tour/default-parameter-values.md:32]:
```suggestion
Ici nous devons écrire `y = 1` car nous omettons l’argument `x`.
```

**Review by julienrf — CHANGES_REQUESTED** [review]

**Comment by SethTisue** [comment]
@apointeau care to respond to the review feedback?

**Comment by julienrf** [comment]
Closing for inactivity. Please re-open it if you want to continue this work.

**Pull request closed without merging**